cli/man/transact-command-get-state.1.md

```diff
@@ -0,0 +1,94 @@
+% TRANSACT-COMMAND-GET-STATE(1) Cargill, Incorporated | Transact Commands
+<!--
+  Copyright 2018-2021 Cargill Incorporated
+  Licensed under Creative Commons Attribution 4.0 International License
+  https://creativecommons.org/licenses/by/4.0/
+-->
+
+NAME
+====
+
+**transact-command-get-state** — Submits a Sabre transaction to request a state 
+read
+
+SYNOPSIS
+========
+| **transact command get state** \[**FLAGS**\] \[**OPTIONS**\]
+
+DESCRIPTION
+===========
+This command submits a Sabre transaction to request a state read of the
+addresses given.
+
+This command assumes the distributed ledger's REST API supports Cylinder
+JWT authentication.
+
+FLAGS
+=====
+`-h`, `--help`
+: Prints help information
+
+`-q`, `--quiet`
+: Decrease verbosity (the opposite of -v). When specified, only errors or
+  warnings will be output.
+
+`-V`, `--version`
+: Prints version information
+
+`-v`
+: Increases verbosity (the opposite of -q). Specify multiple times for more
+  output.
+
+OPTIONS
+=======
+`-k, --key PRIVATE-KEY-FILE`
+: Specifies the full path to the private key file. The key will be used to
+  sign the batches as well as generate a JWT for authentication.
+
+`--target TARGET`
+: Node URL to submit batches to. The URL should include all of the information
+  required to append `/batches` to the end.
+
+`--address ADDRESS`
+: State address of the state to be read.
```

Comment:
Can multiple addresses be supplied here? If multiple may be supplied, would make sense to reference that fact here.  (Same goes for the show command if multiple addresses can be supplied)

Reply:
Updated this to include info about multiple addresses